lightstep/resource_dashboard_test.go

```diff
@@ -187,6 +212,19 @@ resource "lightstep_dashboard" "test" {
 					resource.TestCheckResourceAttr(resourceName, "chart.0.query.0.dependency_map_options.0.map_type", "operation"),
 				),
 			},
+			{
```

Comment:
As someone recently debugging some of these tests, it might be nicer to split out this new test into it's own TestAccDashboardTemplateVariables test (or something similar) rather than including it as a step in a broader test.  Makes it easier to run the test in isolation if nothing else.